templates/Includes/CommentsInterface_singlecomment.ss

```diff
@@ -33,7 +33,9 @@
 				<% end_if %>
 			</div>
 			<% if $RepliesEnabled && $canPostComment %>
-				<a class="comment-reply-link" href="#{$ReplyForm.FormName}"><% _t('CommentsInterface_singlecomment_ss.REPLYTO','Reply to') %> $AuthorName.XML</a>
+				<button class="comment-reply-link" type="button" aria-controls="$ReplyForm.FormName" aria-expanded="false">
++					<% _t('CommentsInterface_singlecomment_ss.REPLYTO','Reply to') %> $AuthorName.XML
++				</button>
```

Comment:
Looks great, and thank you for contributing! Could you please remove the `+` at the start of these two lines? Then we're good to merge